.cicd/platforms/pinned/centos-8-pinned.dockerfile

```diff
@@ -17,6 +17,8 @@ RUN curl -s https://packagecloud.io/install/repositories/rabbitmq/erlang/script.
     yum install -y erlang
 RUN curl -s https://packagecloud.io/install/repositories/rabbitmq/rabbitmq-server/script.rpm.sh | bash && \
     yum install -y rabbitmq-server
+# install request and requests_unixsocket module
+RUN pip3 install requests && pip3 install requests_unixsocket
```

Comment:
This can just be
`pip3 install requests requests_unixsocket`

Reply:
Done.